proto/defs.bzl

```diff
@@ -12,6 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("//proto/private:native.bzl", "NativeProtoInfo", "native_proto_common")
```

Comment:
Do you need this trick, doesn't 

```
_NativeProtoInfo = ProtoInfo
ProtoInfo = _NativeProtoInfo
```

or something similar work?

It's also a little useless, because you can't exactly catch all the uses of `ProtoInfo` and `proto_common` in any way, can you?

Reply:
Nope, unfortunately that throws
```
ERROR: .../proto/defs.bzl:46:20: global variable 'ProtoInfo' is referenced before assignment.
ERROR: .../proto/defs.bzl:47:13: global variable '_NativeProtoInfo' is referenced before assignment.
ERROR: error loading package 'tests': Extension file 'proto/defs.bzl' has errors
```

You're right that there is no way to force users to load these symbols from rules_proto, but it doesn't hurt to export them already :). We can think of a whitelisting mechanism later.
(It's mostly `ProtoInfo` that will require migration. `proto_common` is empty right now, so I don't think anyone is using it. See docs added in bazelbuild/bazel#8923)

Reply:
Okay then.I'll merge this. Harm, it certainly doesn't do.